services/wallet/service.go

```diff
@@ -190,6 +214,19 @@ func SetupService(ctx context.Context) (context.Context, *Service) {
 		ctx = context.WithValue(ctx, appctx.CustodianRegionsCTXKey, custodianRegions)
 	}
 
+	s.jobs = []srv.Job{
+		{
+			Func:    s.RunVerifiedWalletWorker,
+			Cadence: 15 * time.Second,
```

Comment:
so every 15 seconds, we kick off one goroutine which runs forever the "SendVerifiedWalletOutbox" command, each run will lock one entry in the outbox, perform the api calls commit the record.  then run again, until we run out of rows.

What happens if there are more entries than 15 seconds worth of running?

Reply:
This job is passed [here](https://github.com/brave-intl/bat-go/blob/aaccb9487e5423c4c447c5f4a920b0c93b23bfea/services/cmd/serve.go#L112) which in turn in turn calls a single go routine on the JobWorker, the job worker will run the verified wallet job in an infinite loop [here](https://github.com/brave-intl/bat-go/blob/aaccb9487e5423c4c447c5f4a920b0c93b23bfea/libs/service/service.go#L29). 

The 15 seconds acts as a 'backoff ' so if we error or there are no rows otherwise it will keep churning through the [rows](https://github.com/brave-intl/bat-go/pull/1694/files#diff-9751b7432d0f485eae824a65612112872a8159b20c5ef1dd38484ec015dc4a84R621). 

In 2.5 we will only link once so when there are no rows we will hit the db 4 time in 1 minute when there are no rows.